### PR TITLE
Improve the panic code about the rootcoord/session/rocksmq

### PR DESCRIPTION
**Pull request by SimFG**
/kind improvement

### CODE REVIEW

File: internal/rootcoord/dml_channels.go [hunk 1]
```diff
@@ -278,12 +283,10 @@ func (d *dmlChannels) broadcastMark(chanNames []string, pack *msgstream.MsgPack)
 
 func (d *dmlChannels) addChannels(names ...string) {
 	for _, name := range names {
-		v, ok := d.pool.Load(name)
-		if !ok {
-			log.Error("invalid channel name", zap.String("chanName", name))
-			panic("invalid channel name: " + name)
+		dms, err := d.getMsgStreamByName(name)
+		if err != nil {
+			continue
```
Comment (xiaofan-luan) [inline internal/rootcoord/dml_channels.go:288]: add one error message here?

File: internal/util/sessionutil/session_util.go [hunk 2]
```diff
@@ -319,11 +319,16 @@ func (s *Session) getSessionKey() string {
 }
 
 func (s *Session) initWatchSessionCh() {
-	getResp, err := s.etcdCli.Get(context.Background(), s.getSessionKey())
-	if err != nil {
-		panic(err)
+	for {
```
Comment (xiaofan-luan) [inline internal/util/sessionutil/session_util.go:322]: use retry.do()

File: internal/rootcoord/dml_channels.go (hunk 1 above)
Comment (SimFG) [reply internal/rootcoord/dml_channels.go:288]: the error msg has been add in the `getMsgStreamByName` method.
image

**Comment by SimFG** [comment]
rerun ut

**Pull request merged**